go/channelling/turnservice_manager.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Spreed WebRTC.
+ * Copyright (C) 2013-2016 struktur AG
+ *
+ * This file is part of Spreed WebRTC.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package channelling
+
+import (
+	"log"
+	"sync"
+	"time"
+
+	"github.com/strukturag/spreed-turnservicecli/turnservicecli"
+)
+
+type TURNServiceManager interface {
+	TurnDataCreator
+}
+
+type turnServiceManager struct {
+	sync.Mutex
+	pleaders map[uint64]Sender
```

Comment:
Please add a comment what is stored in `pleaders` (like `Mapping of clients waiting to receive TURN data.`). Not everybody might know what `pleaders` means ;-)